File: vortexdbg-api/src/main/java/com/vortexdbg/memory/StackMemory.java
package com.vortexdbg.memory;

import com.vortexdbg.pointer.VortexdbgPointer;
import com.vortexdbg.serialize.Serializable;

public interface StackMemory extends Serializable {

    VortexdbgPointer writeStackString(String str);
    VortexdbgPointer writeStackBytes(byte[] data);

}

File: vortexdbg-api/src/main/kotlin/com/vortexdbg/memory/StackMemory.kt
package com.vortexdbg.memory

import com.vortexdbg.pointer.VortexdbgPointer
import com.vortexdbg.serialize.Serializable

interface StackMemory : Serializable {

    fun writeStackString(str: String): VortexdbgPointer
    fun writeStackBytes(data: ByteArray): VortexdbgPointer

}
